src/Attestation/Attestation/ChangeLog.md

```diff
@@ -21,6 +21,9 @@
 
 ## Upcoming Release
 
+## Version 0.1.3
```

Comment:
You don't need to add new version in a changelog. The only thing you need to do for changelog.md is add the changes in `upcoming` session. We will update the version for you when publishing the package.

Reply:
remove 
## Version 0.1.3
* Update .Net SDK with newer version